promscale/visualize-data/grafana.md

```diff
@@ -0,0 +1,67 @@
+# Connect Promscale and Grafana
+
+To connect Grafana with Promscale, you should have Grafana running. If you do not have Grafana installed, follow the [Grafana installation docs][grafana-install].
+
+This section shows you how to connect Promscale as a Prometheus, Jaeger and PostgreSQL data sources in
+[Grafana][grafana-homepage].
+
+<procedure>
+
+## Promscale as Prometheus datasource
+
+1.  Navigate to `Configuration → Data Sources → Add data source → Prometheus`.
+1.  Configure the data source settings:
+    *   In the `Name` field, type `Promscale-metrics`.
+    *   In the `URL` field, type `http://<PROMSCALE-IP-ADDR>:9201`, using the IP
+        address of your Promscale instance.
+    *   Use the default values for all other settings.
+
+</procedure>
+
+Once you have configured Promscale as a Prometheus data source in Grafana, you can create panels that are populated with data using PromQL
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/images/misc/getting-started-with-promscale-grafana-dashboard.png" alt="Sample output for PromQl query"/>
+
+<procedure>
+
+## Promscale as Jaeger datasource
+
+1.  Navigate to `Configuration → Data Sources → Add data source → Jaeger`.
+1.  Configure the data source settings:
+    *   In the `Name` field, type `Promscale-traces`.
+    *   In the `URL` field, type `http://<PROMSCALE-IP-ADDR>:9201`, using the IP
+        address of your Promscale instance.
+    *   Use the default values for all other settings.
+
+</procedure>
+
+You can now filter and view traces stored in Promscale using Grafana. To visualize your traces, go to the “Explore” section of Grafana. You will be taken to the traces filtering panel.
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/images/misc/grafana-jaeger-query-results.png" alt="Sample output for Jaeger query filter in Grafana"/>
+
+<procedure>
+
+## Promscale as PostgreSQL datasource
+
+1.  Navigate to `Configuration → Data Sources → Add data source → PostgreSQL`.
+1.  Configure the data source settings:
+    *   In the `Name` field, type `Promscale-SQL`.
+    *   In the Host field, type `<host>:<port>`, where host and port need to be 
+        obtained from the service url you copied when you created the Timescale cloud service. The format of that url is `postgresql://[user[:password]@][host][:port][/dbname][?param1=value1&...]`
+    *   In the `Database` field, type the dbname from the service url.
+    *   In the User and Password fields, type the user and password from the service.
+    *   Change the `TLS/SSL Mode` to require as the service url by default contains 
+        the TLS mode as required.
+    *   Change the `TLS/SSL Method` File system path.
+    *   Use the default values for all other settings.
+    *   In the PostgreSQL details section enable the TimescaleDB option.
```

Comment:
```suggestion
    *   In the PostgreSQL details section, check the `TimescaleDB` option.
```

Reply:
I prefer `enable` here as the enabling TimescaleDB option isn't a checkbox it's a kind of bar icon that needs to be clicked to enable. So the word enable represents the user that it needs to be turned on.